apps/web/src/components/Disclosure.tsx

```diff
@@ -15,9 +14,14 @@ export const Disclosure: React.FC<DisclosureProps> = ({ buttonText, content }) =
           <HeadlessDisclosure.Button
             className={'bg-bcBlueBar rounded-b-none flex justify-between w-full py-2'}
           >
+            <img
```

Comment:
Its size is a little bigger than design. Why don't you do 'copy as svg' with fi:chevron-up instead of Vector element?